spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/servlet/error/DefaultErrorAttributes.java

```diff
@@ -94,7 +102,7 @@ private void storeErrorAttributes(HttpServletRequest request, Exception ex) {
 	private void addStatus(Map<String, Object> errorAttributes,
 			RequestAttributes requestAttributes) {
 		Integer status = getAttribute(requestAttributes,
-				"javax.servlet.error.status_code");
+				RequestDispatcher.ERROR_STATUS_CODE);
```

Comment:
Against `master`, please. But only if you're feeling particularly generous; it's not the most exciting of changes :)